src/user/models/kyc/dto/kyc-user-data.dto.ts

```diff
@@ -47,6 +47,16 @@ export class KycUserDataDto {
   @Type(() => EntityDto)
   country: Country;
 
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsString()
+  mail: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsString()
+  phone: string;
```

Comment:
That it will not be overwritten by accident?
Thats why they are sent via `KycInfo` and are called `blankedPhone` and `blankedMail`.

Reply:
Let's say somebody enters the KYC Data on payment, but had mail and phone already saved. You will then call the /data endpoint and mail and phone are required, but you only have the blanked ones.

Reply:
Hm trying to explain it shorter, because github deleted what i previously wrote.

On payment there will be a functionality to pass users' mail and phone to /kyc and prefill on the kyc edit form. If this is not being used, we have basically two possibilities.

1) let the user reenter his information, maybe it has changed (low probability)
2) make those two properties optional and remove them from the form if blanked information is present.

Reply:
we decided to make everything optional and make additional changes to payment site